bitnami/kibana/templates/deployment.yaml

```diff
@@ -73,6 +73,7 @@ spec:
           securityContext:
             runAsUser: {{ .Values.securityContext.runAsUser }}
           {{- end }}
+          serviceAccountName: {{ .Values.serviceAccount.name }}
```

Comment:
We use a template in order to render the proper serviceAccount name, you can base on the following: https://github.com/bitnami/charts/blob/c10c7f4015d1ed0bf52ed10f9cd0cc6fb7d94910/bitnami/redis/templates/_helpers.tpl#L141